src/engine/specification.md

```diff
@@ -365,6 +369,30 @@ The payload build process is specified as follows:
 
 3. Client software **MAY** stop the corresponding build process after serving this call.
 
+### engine_getPayloadV2
+
+#### Request
+
+* method: `engine_getPayloadV2`
+* params:
+  1. `payloadId`: `DATA`, 8 Bytes - Identifier of the payload build process
+* timeout: 1s
+
+#### Response
+
+* result: `object`
+  - `executionPayloadV1`: [`ExecutionPayloadV1`](#ExecutionPayloadV1)
```

Comment:
Shouldn't this be `ExecutionPayloadV2` ?

Reply:
yes, this was a originally a pre-Shanghai method.  Added getPayloadV3 for Shanghai with `ExecutionPayloadV2`

Reply:
Per the comments below, updated getPayloadV2 to use ExecutionPayloadV2